src/main/eo/org/eolang/collections/list.eo

```diff
@@ -178,6 +178,19 @@
           i
           acc.plus 0
 
+  # Returns index of the last particular item in list.
+  # If the list has no this item, last-index-of returns -1
+  [wanted] > last-index-of
+    ^.reducedi > @
+      -1
+      [acc i item]
+        if. > @
+          eq.
+            item
+            wanted
+          i
+          acc.plus 0
```

Comment:
@mximp referring to the `index-of` implementation, its creator made it so as to cause an explicit copying of the object, so it will not be recomputed many times. So yes, we can probably drop that, so it's just optimization.